src/components/tools/networked-drawing.js

```diff
@@ -204,7 +204,7 @@ AFRAME.registerComponent("networked-drawing", {
     };
 
     const glb = await new Promise(resolve => {
-      exporter.parse(mesh, resolve, {
+      exporter.parse(mesh, resolve, undefined, {
```

Comment:
We should probably reject the promise if the error callback gets called. I assume before it was throwing an exception which would cause `serializeDrawing` to return a rejected promise since it is an async function.

Reply:
Good catch. Updated to reject the promise on error.